earth_enterprise/src/fusion/gepublish/SConscript

```diff
@@ -46,7 +47,7 @@ swig_env['CPPFLAGS'] += ['-fPIC']  # Needed for 64-bit compilation.
 swig_env['CPPFLAGS'] += ['-I%s' % swig_env['python_include']]
 swig_env.Append(SWIGFLAGS = ['-python', '-c++'])
 
-libgepublishmanagerhelper = swig_env.sharedLib('gepublishmanagerhelper',
+libgepublishmanagerhelper = swig_env.SharedLibrary('gepublishmanagerhelper',
```

Comment:
What is this change doing?

Reply:
Ah. Yeah that needs changed back to sharedlib. Sharedlibrary is the scons builder, and sharedLib appears to be our kh version.